content/comm/conferences/index.en.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "R-Ladies Conference Participation"
+menuTitle: "Conference Participation"
+weight: 110
+---
+
+R-Ladies participates in R-Ladies branded conference sessions and events such as
+contributed talks, panel discussions, networking events, poster presentations, and booths to 
+increase awareness of our organization.
+
+## R-Ladies branded conference event
+
+An R-Ladies branded event is one that used the name `R-Ladies` to....
```

Comment:
```suggestion
An R-Ladies branded event is one that used the name `R-Ladies` in its name.
```

Is this what you mean?

Reply:
Yes - I did not know how to define the scope!

Reply:
@maelle it looks like I overwrote your suggested change - sorry, that was not my intention! Modified slightly to use the word `name` less.